Bugses patch 1

Bugses opened the pull request.

Bugses committed:
Update README.md

Bugses committed:
added starter content

ad sasd asd sd

Bugses committed:
moved startercontent

Bugses committed:
Create no skip.txt

brendanehrman1 commented on README.md:
```diff
@@ -1 +1,2 @@
-# GameFall2022
\ No newline at end of file
+# GameFall2022
```
you can change this back now

Bugses replied:
nah

brendanehrman1 reviewed (changes requested):
looks fine, just update readme and delete no skip.txt

Bugses committed:
added the things

Bugses committed:
added spherical gravity + movement fixes

changed the movement from the default settings so that it would work with spherical gravity

Bugses committed:
physics modification + wall and long jumps

Bugses committed:
added animations and non-radial directional gravity
